src/components/authentication/LogIn.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { SigninDetailsApi } from '../../redux/reducer/user';
+
+const LogIn = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState('');
+  const [password, setPassowrd] = useState('');
+  const state = useSelector((state) => state.UserReducer);
+  const { loggedIn } = state;
+  // const [signedInSuccess, setSignedInSuccess] = useState(loggedIn);
```

Comment:
Please remove commented code as it clutters your work.

Reply:
Thank @Alfred-KInyua  to mention the commented code.